src/test/cypress/support/pageobjects/assessment/CourseAssessmentDashboardPage.ts

```diff
@@ -12,7 +12,7 @@ export class CourseAssessmentDashboardPage {
     }
 
     showTheComplaint() {
-        cy.get('.btn-primary').should('contain.text', 'Show the complaint').click();
+        cy.get('#show-complaint').should('contain.text', 'Show the complaint').click();
```

Comment:
Verifying the text of the complaint button only makes the test more unstable in my opinion
```suggestion
        cy.get('#show-complaint').click();
```